hyundai_kia_connect_api/KiaUvoApiAU.py

```diff
@@ -657,18 +664,6 @@ def _update_vehicle_drive_info(self, vehicle: Vehicle, state: dict) -> None:
         vehicle.power_consumption_30d = get_child_value(state, "consumption30d")
         vehicle.daily_stats = get_child_value(state, "dailyStats")
 
-    def _get_cached_vehicle_state(self, token: Token, vehicle: Vehicle) -> dict:
```

Comment:
This was only used in `update_vehicle_with_cached_state` so I just inlined it instead to make switching over ccs2 vehicles easier